ProcessMaker/processmaker
# FOUR-7854: Fix Ellipse styling for Process Templates

## Issue & Reproduction Steps

Currently, the template configurations dropdown menu is displayed with the first template cut off. To reproduce the issue, follow these steps:

1. Create a single process template.
2. Select the ellipse menu.

**Current Behavior**

The menu buttons are cut off and not fully visible.

**Expected Behavior**

All menu buttons should be fully visible and accessible to the user.

## Solution
To resolve this issue, we needed to add the missing styling to the process-templates-table to properly display the dropdown menu.

## How to Test
To test the solution, please follow the reproduction steps above after the styling update has been implemented. You should be able to see all the menu buttons without any issues.

## Code Review Checklist
